A5_2practice_2.java

```diff
@@ -0,0 +1,55 @@
+import java.util.Scanner;
+public class A5_2practice_2 {
+
+	public static void main(String[] args) {
+		
+		boolean isPrime;
+		boolean isPerfect;
+	
+		Scanner scan= new Scanner(System.in);
+		System.out.println("Enter any number:");
+		//capture the input in an integer
+		int n=scan.nextInt();
+	       
+	if ( isPerfect= true){
+			System.out.println(n  +" is a perfect number");
+	        }
+		
+	     
+	    if (isPrime = true){ 
+	            System.out.println(n +" is a prime number");
+        }
+	}
+		
+		public static boolean isPrime(int n, boolean isPrime) {
+			
+			if (n %2!=0) {   
+			return true;
+		}
+			return isPrime;
+		}
+
+
+	public static boolean isPerfect(int n, boolean isPerfect)

```

Comment:
this is good. Can you just cleanup the indentation and brackets so that it's easier to read

Reply:
also you should just pass in an int, not the boolean isPerfect.

Reply:
Thank you Samuel! 
I'll work on it.